store/views.py

```diff
@@ -77,6 +99,42 @@ def loanBookView(request):
 @csrf_exempt
 @login_required
 def returnBookView(request):
-    pass
+    post_data = request.POST
+    try:
+        bookCopy = BookCopy.objects.get( id__exact=post_data['bid'], borrower=request.user )
```

Comment:
You are directly accessing POST data without checking if it even exists. This may lead to server crash if a user access this endpoint with invalid request data. The good behavior would have been to throw a client error (400), rather than server error (500).